test/Sign.Core.Test/TestInfrastructure/TemporaryEnvironmentPath.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE.txt file in the project root for more information.
+
+namespace Sign.Core.Test
+{
+    internal sealed class TemporaryEnvironmentPath : IDisposable
```

Comment:
`TemporaryEnvironmentPathOverride`? I think that would better convey what the class does.